lms/static/sass/_build-discussion.scss

```diff
@@ -0,0 +1,16 @@
+// From _build-lms-v1.scss
```

Comment:
@andy-armstrong Thanks for the info. This is a refactoring I'd rather tackle in a separate PR: It's not a requirement for the current goal of having `DiscussionXBlock` take care of loading assets it depends on (as specified by [YONK-382](https://openedx.atlassian.net/browse/YONK-382)).